src/Restore.php

```diff
@@ -122,7 +124,11 @@ public function restoreConfigs(array $skipComponents = []): void
                     ConfigurationFilter::removeOauthAuthorization($configurationData->configuration)
                 );
                 if (isset($configurationData->state)) {
-                    $configuration->setState($configurationData->state);
+                    $configurationState = new ConfigurationState();
+                    $configurationState->setComponentId($componentWithConfigurations['id']);
+                    $configurationState->setConfigurationId($componentConfiguration['id']);
+                    $configurationState->setState($configurationData->state);
+                    $components->updateConfigurationState($configurationState);
```

Comment:
Asi by som to presunul az pod `updateConfiguration`, takto buildujes 2 requesty naraz, co moze niekoho pomylit.

Reply:
ok, může být :)

Reply:
https://github.com/keboola/php-kbc-project-restore/pull/26/commits/9a4df7000fc108e7417c601b2c4fd01a257ccbf8

Reply:
:+1: